NeoML/include/NeoML/Dnn/Layers/RowwiseOperationChainLayer.h

```diff
@@ -48,6 +48,7 @@ class NEOML_API CRowwiseOperationChainLayer : public CBaseLayer {
 	// MathEngine descriptors of operations in chain
 	CArray<CRowwiseOperationDesc*> operationDescs;
```

Comment:
Does it own that descriptors? Could be `CPointerArray`?

Reply:
This line is from old version of PR #1090, it will be deleted after merge and update from master

Reply:
fixed